test/xUnit/csharp/test_DefaultVisit.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
```

Comment:
Perhaps we will add more tests for Ast-s so I'd name the file like `test_AstDefaultVisit`.